listenbrainz/webserver/static/css/recommendation-page.less

```diff
@@ -0,0 +1,61 @@
+#recommendations {
+  &:extend(#listens);
+  padding: 10px;
+
+  > .recommendation-card {
+    padding: 15px;
+    margin-bottom: 7px;
+    max-height: 72px;
```

Comment:
We'll have to review this before deploying. Spacing looks off to me:
<img width="701" alt="Capture d’écran 2020-10-15 à 11 37 44" src="https://user-images.githubusercontent.com/6179856/96105947-e904d700-0eda-11eb-8ba9-4c698c463b79.png">